tests/suites/0_stateless/02_0012_function_datetimes.sql

```diff
@@ -18,7 +18,7 @@ select today() + 1 = tomorrow();
 select tomorrow() - today() = 1;
 
 select toYYYYMM(toDateTime(1630320462));
-select toYYYYMM(today() - 30) = toYYYYMM(today()) - 1;
-select toYYYYMM(today() + 30) = toYYYYMM(today()) + 1;
-select toYYYYMM(today()) - toYYYYMM(today() - 30) = 1;
-select toYYYYMM(today() + 30) - toYYYYMM(today()) = 1;
+select toYYYYMM(today() - 31) = toYYYYMM(today()) - 1;
```

Comment:
That's not correct too. What if today is `03-01`,  `toYYYYMM(today() - 31)` will be `toYYYYMM(today()) - 2`

Reply:
Yeah, you are right.
For the case of 03-01, all the four eq expression may return false.
As datafuse does not support inserting date16/date32/datetime32 sql like following now.
````
mysql> CREATE TABLE test(a UInt8, b date) engine = Memory;
Query OK, 0 rows affected (0.01 sec)

mysql> desc test;
+-------+--------+------+
| Field | Type   | Null |
+-------+--------+------+
| a     | UInt8  | NO   |
| b     | Date16 | NO   |
+-------+--------+------+
2 rows in set (0.00 sec)

mysql> insert into test(a, b) values(1, toDate(1630320462));
ERROR 1105 (HY000): Code: 10, displayText = create_deserializer does not support type 'Date16'.
mysql> insert into test(a, b) values(1, '2021-08-10');
ERROR 1105 (HY000): Code: 10, displayText = create_deserializer does not support type 'Date16'.
```
How can I add the stateless test? Please give some idea.

Reply:
> As datafuse does not support inserting date16/date32/datetime32 sql like following now.

Yes, we will add this in another pr.
Cureently 

```
select toUInt32(now()) returns `1630833797`
select toUInt16(today()) returns `18875`
```
How about  

```
select toYYYYMM(toDateTime(1630833797)  )  =  202109;
select toYYYYMM(toDate(18875)  )  =  202109;

```

Reply:
Ok.